### PR TITLE
Remove pic2card and experimental source

### LINKED ISSUE #6789: Remove unneeded pic2card source

### PR DESCRIPTION
**Pull request by paulcam206**
# Related Issue

Fixes #6789 

# Description

Simple removal of pic2card and experimental source since it should not live in our codebase.

# How Verified

* local build and test of site

###### Microsoft Reviewers: [Open in CodeFlow](https://portal.fabricbot.ms/api/codeflow?pullrequest=https://github.com/microsoft/AdaptiveCards/pull/6790)

### CODE REVIEW

**Comment by licanhua** [comment]
Can you keep source/pic2card/README.md, and state that that project is deprecated and source code is in https://github.com/microsoft/AdaptiveCards/releases/tag/21.09?

**Comment by licanhua** [comment]
workflow, bug template and adaptivecards/readme need to update too

image
image

File: source/pic2card/ui/README.md [hunk 1]
```diff
@@ -1,29 +0,0 @@
-# ann-app
-
```
Comment (licanhua) [inline source/pic2card/ui/README.md:2]: keep this file and mark it as deprecated.
update /readme.md
update bug template, workflow

image
image

File: source/pic2card/README.md [hunk 2]
```diff
@@ -1,152 +0,0 @@
-# Pic2Card
-![pic2card-build](https://github.com/Microsoft/AdaptiveCards/workflows/pic2card-build/badge.svg)
```
Comment (licanhua) [inline source/pic2card/README.md:2]: keep this file and update all links related to Pic2Cards on /

**Review by licanhua — CHANGES_REQUESTED** [review]

**Comment by almedina-ms** [comment]
With the removal of this project and the vs extension which were provided by the community, is there some plan in place to move these tools into a different repository?

**Comment by paulcam206** [comment]
> With the removal of this project and the vs extension which were provided by the community, is there some plan in place to move these tools into a different repository?

The VS extension was already shipping out of a different repository, so there's really no action to take. I think we would want to partner with folks to spotlight community work (and be available to provide guidance or support if they'd like), but it's not really appropriate to host external tools here...

File: source/pic2card/ui/README.md (hunk 1 above)
Comment (paulcam206) [reply source/pic2card/ui/README.md:2]: I've updated `/README.md`, but I don't want to leave breadcrumbs around the repo of projects that are no longer hosted here (see, e.g. the old NodeJS visualizer codebase).

**Review by almedina-ms — APPROVED** [review]

**Review by licanhua — APPROVED** [review]

**Pull request merged**